### DISCUSSION TITLE
Improve UX

### DISCUSSION THREAD

**Discussion by alanski66** [lukevella/rallly#1203] [Ideas]
I use to use this couple of years ago to organise who was coming to basketball so thought it would be great to organise a littel event.
No one has been able to use it on mobile to choose their options for yes, no, maybe are pre-filled as no and grayed out. Even on desktop i struggled to realise the hidden UX of clicking the icon to rotate my choice. With deepest respect this is not good. I love the product but right now its intuitively not working.

**Comment by lukevella** [comment]
Appreciate the feedback. Is it the two clicks for "If need be" that you found unintuitive? Do you have thoughts on what can be done to improve it?
